src/main/java/appeng/container/slot/OptionalSlotFake.java

```diff
@@ -69,12 +69,7 @@ public boolean isSlotEnabled()
 		return this.host.isSlotEnabled( this.groupNum );
 	}
 
-	public boolean renderDisabled()
-	{
-		return this.isRenderDisabled();
-	}
-
-	private boolean isRenderDisabled()
+	public boolean isRenderDisabled()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1161)